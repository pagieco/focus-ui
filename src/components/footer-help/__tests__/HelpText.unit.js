import { shallowMount } from '@vue/test-utils';
import { FooterHelp } from '../index';

describe('HelpText', () => {
  it('is a valid vue component', () => {
    expect(FooterHelp).toBeAComponent();
  });

  it('matches the snapshot', () => {
    expect(shallowMount(FooterHelp).element).toMatchSnapshot();
  });
});
