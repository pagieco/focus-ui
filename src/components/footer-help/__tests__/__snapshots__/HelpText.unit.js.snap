// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpText matches the snapshot 1`] = `
<div
  class="footer-help"
>
  <div
    class="footer-help__content"
  >
    <div
      class="footer-help__icon"
    >
      <icon-stub
        name="question"
      />
    </div>
     
    <div
      class="footer-help__text"
    />
  </div>
</div>
`;
